deb_from_installation.sh

```diff
@@ -1,5 +1,16 @@
 #!/usr/bin/env sh
+set -x
+
+command -v dpkg-deb >/dev/null
+if [ $? -eq 0 ]
+then
```

Comment:
if you change -eq to -ne you can remove the empy then.

Reply:
done, thanks!